apps/api/package.json

```diff
@@ -124,6 +124,7 @@
     "@types/bull": "^3.15.8",
     "@types/chai": "^4.2.11",
     "@types/express": "4.17.7",
+    "@types/got": "^9.6.12",
```

Comment:
@djabarovgeorge I think this is not needed right now? or we still use got in other partS?

Reply:
oh my, I was sure i removed it.

Reply:
Oh it is the types one that i missed. 
Good catch!